Modules/IO/JPEG/include/itkJPEGImageIO.h

```diff
@@ -49,8 +52,8 @@ class ITKIOJPEG_EXPORT JPEGImageIO:public ImageIOBase
   itkTypeMacro(JPEGImageIO, ImageIOBase);
 
   /** Set/Get the level of quality for the output images. */
-  itkSetMacro(Quality, int);
-  itkGetConstMacro(Quality, int);
+  virtual void SetQuality(int _JPEGQuality) { this->SetCompressionLevel(_JPEGQuality); }
```

Comment:
That would need to extra documentation. As a user I would find it very hard to understand that a compression ratio is now an image quality (from the top of my head jpeg quality here is a PSNR right?).